src/olympia/accounts/serializers.py

```diff
@@ -174,7 +174,7 @@ class AccountSuperCreateSerializer(serializers.Serializer):
     username = serializers.CharField(required=False)
     email = serializers.EmailField(required=False)
     fxa_id = serializers.CharField(required=False)
-    group = serializers.ChoiceField(choices=group_rules.items(),
+    group = serializers.ChoiceField(choices=dict(group_rules.items()),
```

Comment:
Shouldn't it be `list` instead ?

Reply:
yeah... I was thinking "group_rules is a dict so it'll be a dict".  It's weird that none of the tests picked it up though.

Reply:
https://github.com/mozilla/addons-server/pull/10485/commits/a081d19e122826bd07c1d9f3afdeedf42f4be308